standup/status/management/commands/mergeuser.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from standup.status.models import StandupUser
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    help = 'Merge two user accounts'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--keep', type=int)
+        parser.add_argument('--delete', type=int)
+        parser.add_argument('-y', '--yes', dest='assume_yes', action='store_true')
+
+    def handle(self, *args, **options):
+        try:
+            user_keep = User.objects.get(id=options['keep'])
+        except User.DoesNotExist:
+            self.stdout.write('User keep=%s does not exist. Exiting.' % options['keep'])
+            return 1
+
+        try:
+            user_delete = User.objects.get(id=options['delete'])
+        except User.DoesNotExist:
+            self.stdout.write('User delete=%s does not exist. Exiting.' % options['delete'])
+            return 1
+
+        def get_status_count(user):
+            try:
+                return StandupUser.objects.get(user=user).statuses.count()
+            except StandupUser.DoesNotExist:
+                return 0
+
+        self.stdout.write('Deleting and merging from:')
+        self.stdout.write('    id:          %s' % user_delete.id)
+        self.stdout.write('    username:    %s' % user_delete.username)
+        self.stdout.write('    email:       %s' % user_delete.email)
+        self.stdout.write('    date_joined: %s' % user_delete.date_joined)
+        self.stdout.write('    statuses:    %s' % get_status_count(user_delete))
+        self.stdout.write('')
+        self.stdout.write('Keeping and merging into:')
+        self.stdout.write('    id:          %s' % user_keep.id)
+        self.stdout.write('    username:    %s' % user_keep.username)
+        self.stdout.write('    email:       %s' % user_keep.email)
+        self.stdout.write('    date_joined: %s' % user_keep.date_joined)
+        self.stdout.write('    statuses:    %s' % get_status_count(user_keep))
+        self.stdout.write('')
+
+        if not options['assume_yes']:
+            self.stdout.write('Continue?: Y/n')
```

Comment:
NIT: When I see one of these and one of the letters is capitalized (the Y in this case) I assume that's the default if I hit [enter] w/o typing a letter. That doesn't appear to be how this would work. Not important; just an observation.